### PR TITLE
Allow to run on a dom-element (from domutils.parseDOM)

### PR DESCRIPTION
**Pull request by kesla**
R = @ellell 
T = minor

### CODE REVIEW

**Comment by ellell** [comment]
Travis is still angry, other than that lgtm though!

File: index.js [hunk 1]
```diff
@@ -31,14 +39,14 @@ function hyperfast (html, params) {
...
   });
 
-  var body = elems.map(function (elm) {
-    return domutils.getOuterHTML(elm);
-  }).join('');
+  // unwrap dom, if needed
+  if (wrapper) {
+    // elems.forEach(function (elem) {
+    //   elem.parent = null;
+    // });
```
Comment (dey-dey) [inline index.js:46]: is this supposed to be commented?

**Comment by dey-dey** [comment]
also tests?

File: index.js (hunk 1 above)
Comment (kesla) [inline index.js:46]: ooohhh shit. haha

Comment (dey-dey) [inline index.js:46]: ;>

**Pull request merged**